app/metrics.yaml

```diff
@@ -1776,3 +1776,70 @@ app_theme:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+installation:
+  campaign:
+   type: string
+   send_in_pings:
+     - installation
+   description: >
+     Campaign
```

Comment:
Please be more explicit and descriptive about what this is, its format and its purpose.

Reply:
In regards to this and all the other improper descriptions in the metrics.yaml & pings.yaml: 
- I'm afraid [the issue](https://github.com/mozilla-mobile/fenix/issues/7295) doesn't specify what those adjust properties represent, so I did not know what to write in their descriptions. I was hoping a reviewer would help.

Reply:
I think @boek or somebody else on the Fenix team could probably provide more insights with this :)